extensions/wikia/DesignSystem/i18n/build.js

```diff
@@ -0,0 +1,51 @@
+var fs = require('fs'),
+	path = require('path'),
+	rootDir = process.cwd() + '/node_modules/design-system-i18n/i18n',
+	filename = 'design-system.json',
+	destDir = './i18n',
+	// keep in sync with DesignSystem/DesignSystemHelper.class.php
+	messageParamsMapping = {
```

Comment:
I like the idea but mapping in both places isn't 1 to 1. Currently it's only one message so how about rethinking this if needed during our work on Global Nav?